cilium/cmd/preflight_k8s_valid_cnp.go

```diff
@@ -70,13 +75,22 @@ func validateCNPs() error {
 		log.WithError(err).Fatal("Unable to connect to Kubernetes apiserver")
 	}
 
+	restConfig, err := k8s.CreateConfig()
+	if err != nil {
+		return fmt.Errorf("Unable to create rest configuration for k8s CRD: %s", err)
```

Comment:
`error` type is not a string. Prefer new `%w` verb or use `err.Error()` function to avoid formatting errors.
```suggestion
		return fmt.Errorf("Unable to create rest configuration for k8s CRD: %w", err)
```